src/core/infrastructure/command.d.ts

```diff
@@ -0,0 +1,7 @@
+export declare class Command {
+   constructor(context: object);
+
+   execute(): void;
```

Comment:
please fix formatting, also execute and canExecute can accept arguments, is'it possible to declare that?